install/samples/lazyload/slimeboot_cluster_accesslog.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: config.netease.com/v1alpha1
+kind: SlimeBoot
+metadata:
+  name: lazyload
+  namespace: mesh-operator
+spec:
+  image:
+    pullPolicy: Always
+    repository: docker.io/slimeio/slime-lazyload
+    tag: master-ecadccb-dirty_8d0e997
+  module:
+    - name: lazyload
+      enable: true
+      fence:
+        wormholePort: # replace to your application service ports, and extend the list in case of multi ports
+          - "80"
+          - "9080"
+        namespace: # replace to your service's namespace which will use lazyload, and extend the list in case of multi namespaces
+          - default
+      global:
+        misc:
+          global-sidecar-mode: cluster # inform the lazyload controller of the global-sidecar mode, default is namespace
+          metric_source_type: accesslog # indicate the metric source
+  component:
+    globalSidecar:
+      enable: true
+      type: cluster # inform the slime-boot operator of the global-sidecar mode
+      sidecarAutoInject:
+        enable: true # must be true
+        mode: pod # if type = cluster, can only be "pod"; if type = namespace, can be "pod" or "namespace"
```

Comment:
This `pod`/`namespace` is not so epistemic and not sufficiently commented

Reply:
done

Reply:
comments as below
        # mode definition:
        # "pod": sidecar auto-inject on pod level, need provide labels for injection
        # "namespace": sidecar auto-inject on namespace level, no need to provide labels for injection
        # if global_sidecar_mode is cluster, global-sidecar will be deployed in slime namespace, which does not enable auto-inject on namespace level, mode can only be "pod".
        # if global_sidecar_mode is namespace, depending on the namespace definition, mode can be "pod" or "namespace".